packages/redux-requests-react/types/index.d.ts

```diff
@@ -136,20 +140,21 @@ export function useSubscription<SC extends SubscriptionCreator = any>(props: {
 
 export function useDispatchRequest(): DispatchRequest;
 
-interface RequestsProviderProps {
+type RequestsProviderProps =
+  ({ requestsConfig: HandleRequestConfig; store?: never; }
+  | { store: Store; requestsConfig?: never; })
```

Comment:
Nice idea, but would it be possible to take other props redundant when `store` is defined? Like `extraReducers` and `getMiddleware`

Reply:
Sorry i was so long to fix, are my new changes what you meant?

Reply:
There is nothing to apologize for, this is exactly what I meant and thank you very much for your time and for this amazing PR! I am merging this and releasing right away!